pyramid/scripts/pserve.py

```diff
@@ -48,92 +47,106 @@ class PServeCommand(object):
     """
     default_verbosity = 1
 
-    parser = optparse.OptionParser(
-        usage,
+    parser = argparse.ArgumentParser(
+        usage="%(prog)s config_uri [var=value]",
```

Comment:
usage is generated by argparse... was it not acceptable? Why is it being overridden?